Tasks/Romanychev/Task2/dictionary.py

```diff
@@ -0,0 +1,25 @@
+time_word_mapping = {
+    0: ('', 'минут', 'двенадцатого', '', '', 'двенадцать', 'часов'),
+    1: ('одна', 'минута', 'первого', 'одной', 'минуты', 'час', ''),
+    2: ('две', 'минуты', 'второго', 'двух', 'минут', 'два', 'часа'),
+    3: ('три', 'минуты', 'третьего', 'трех', 'минут', 'три', 'часа'),
+    4: ('четыре', 'минуты', 'четвертого', 'четырех', 'минут', 'четыре', 'часа'),
+    5: ('пять', 'минут', 'пятого', 'пяти', 'минут', 'пять', 'часов'),
+    6: ('шесть', 'минут', 'шестого', 'шести', 'минут', 'шесть', 'часов'),
+    7: ('семь', 'минут', 'седьмого', 'семи', 'минут', 'семь', 'часов'),
+    8: ('восемь', 'минут', 'восьмого', 'восьми', 'минут', 'восемь', 'часов'),
+    9: ('девять', 'минут', 'девятого', 'девяти', 'минут', 'девять', 'часов'),
+    10: ('десять', 'минут', 'десятого', 'десяти', 'минут', 'десять', 'часов'),
+    11: ('одиннадцать', 'минут', 'одиннадцатого', 'одиннадцати', 'минут', 'одиннадцать', 'часов'),
+    12: ('двенадцать', 'минут', 'двенадцатого', 'двенадцати', 'минут', 'двенадцать', 'часов'),
+    13: ('тринадцать', 'минут', '', 'тринадцати', 'минут', '', ''),
+    14: ('четырнадцать', 'минут', '', 'четырнадцати', 'минут', '', ''),
+    15: ('пятнадцать', 'минут', '', 'пятнадцати', 'минут', '', ''),
+    16: ('шестнадцать', 'минут', '', '', '', '', ''),
+    17: ('семнадцать', 'минут', '', '', '', '', ''),
+    18: ('восемнадцать', 'минут', '', '', '', '', ''),
+    19: ('девятнадцать', 'минут', '', '', '', '', ''),
+    20: ('двадцать', 'минут', '', '', '', '', ''),
+    30: ('тридцать', '', '', '', '', '', ''),
+    40: ('сорок', 'минут', '', '', '', '', '')
+}
```

Comment:
this is great, but still it has the duplicates of the "minute" words) but it's not a big deal actually